docs/example.md

```diff
@@ -130,7 +130,7 @@ conf = ConnectionConfig(
     MAIL_SERVER = "your mail server",
     MAIL_TLS = True,
     MAIL_SSL = False,
-    TEMPLATE_FOLDER='./email '
+    TEMPLATE_FOLDER = Path(__file__).parent / 'templates',
```

Comment:
hi, thank you for your PR. I think after update these changes will break the projects who used our module. Can you add backward compatibility?